src/app-layout/internal.tsx

```diff
@@ -5,14 +5,15 @@ import React from 'react';
 import { useVisualRefresh } from '../internal/hooks/use-visual-mode';
 import ClassicAppLayout from './classic';
 import { AppLayoutProps, AppLayoutPropsWithDefaults } from './interfaces';
-import { isAppLayoutToolbarEnabled } from './utils/feature-flags';
+import { useAppLayoutToolbarEnabled } from './utils/feature-flags';
```

Comment:
Since I had to put `useVisualRefresh` hook, I had to convert this one as a hook too (but it should not count any performance impact, because we do not use any React hooks API internally anyway)